openedx/core/djangoapps/api_admin/tests/test_models.py

```diff
@@ -64,6 +64,19 @@ def test_unicode(self):
         self.assertIn(self.request.website, request_unicode)  # pylint: disable=no-member
         self.assertIn(self.request.status, request_unicode)
 
+    def test_retire_user_success(self):
+        retire_result = self.request.retire_user(self.user)
+        self.assertTrue(retire_result)
+        self.assertEqual(self.request.company_address, '')
+        self.assertEqual(self.request.company_name, '')
+        self.assertEqual(self.request.website, '')
+        self.assertEqual(self.request.reason, '')
+
+    def test_retire_user_do_not_exist(self):
+        self.user2 = UserFactory()
```

Comment:
Is it worth nailing down that the user isn't in the model?
```
api_requests = ApiAccessRequest.objects.filter(user=self.user2)
self.assertFalse(api_requests.exists())
```

Reply:
Don't think that's needed. We don't have to protect this test case pretending the test setup doesn't exist.